core/commands/add.go

```diff
@@ -232,16 +222,13 @@ You can now refer to the added file in a gateway, like so:
 			return
 		}
 
-		var showProgressBar bool
-		if prgFound {
-			showProgressBar = progress
-		} else if !quiet && !silent {
-			showProgressBar = true
+		if !quiet && !silent {
+			progress = true
```

Comment:
I think this change changes the actual semantics of the code. Because now i can't shut it off if i don't also say quiet or silent. For example, this:

```
ipfs add file --progress=false
```
no longer works, i think.